src/EventEdition/LocationPanel.vala

```diff
@@ -187,6 +189,46 @@ public class Maya.View.EventEdition.LocationPanel : Gtk.Grid {
             debug (error.message);
         }
     }
+
+    private async void find_location (double latitude, double longitude) {
+        if (find_cancellable != null)
+            find_cancellable.cancel ();
+        Geocode.Location location = new Geocode.Location (latitude, longitude);
+        var reverse = new Geocode.Reverse.for_location (location);
+        try {
```

Comment:
I would prefer a preceding blank line for readability

Reply:
Where in particular, before the try?

Reply:
Yes, that's right:

```
       var reverse = new Geocode.Reverse.for_location (location);

       try {
           ...
```